R/plotDataSetList.R

```diff
@@ -987,17 +989,27 @@ Plot.FV.ECDF_Single_Func.DataSetList <- function(dsList, rt_min = NULL, rt_max =
 
   rt_seq <- seq_RT(rt, from = rt_min, to = rt_max, by = rt_step,
                    scale = ifelse(scale.xlog,'log','linear'))
+  
+  if(!attr(dsList[[1]],"maximization"))
+    rt_seq <- rev(rt_seq)
+  
   req(rt_seq)
 
   n_algorithm <- length(dsList)
   colors <- color_palettes(n_algorithm)
 
   funevals.max <- sapply(dsList, function(ds) max(ds$FV, na.rm = T)) %>% max
   funevals.min <- sapply(dsList, function(ds) min(ds$FV, na.rm = T)) %>% min
-
-  x <- seq(funevals.min, funevals.max, length.out = 40)
+  
+  if(!attr(dsList[[1]],"maximization"))
+    x <- seq(log10(funevals.max), log10(funevals.min), length.out = 40) %>% 10**.
```

Comment:
Sometimes, I think I might abuse the pipeline in R (totally my fault...). I think in this case, perhaps it is better to use 10 ^ seq(log10(funevals.max), log10(funevals.min), length.out = 40)